custom/conf/app.ini.sample

```diff
@@ -122,6 +122,8 @@ UNIX_SOCKET_PERMISSION = 666
 ; Local (DMZ) URL for Gitea workers (such as SSH update) accessing web service.
 ; In most cases you do not need to change the default value.
 ; Alter it only if your SSH server node is not the same as HTTP node.
+; If you set PROTOCOL to 'unix', you must comment this variable or set it to
+; 'http://unix/'
 LOCAL_ROOT_URL = %(PROTOCOL)s://%(HTTP_ADDR)s:%(HTTP_PORT)s/
```

Comment:
Can you also update the docs page that references the setting: https://docs.gitea.io/en-us/config-cheat-sheet/

Reply:
I suggest: "If PROTOCOL is 'unix', don't this variable."